solutions/standard/variables.tf

```diff
@@ -208,3 +208,14 @@ variable "auto_scaling" {
   description = "Optional rules to allow the database to increase resources in response to usage. Only a single autoscaling block is allowed. Make sure you understand the effects of autoscaling, especially for production environments. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/solutions/standard/DA-types.md#autoscaling)"
   default     = null
 }
+
+
+##############################################################
+# Backup
+##############################################################
+
+variable "backup_crn" {
```

Comment:
please add validation as we have it in main root https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/variables.tf#L265

Reply:
Wouldn't it be redundant to add validation here since it's already implemented in the main module? I believe we’ve been avoiding validation at higher-level modules when it's handled in the underlying module.

Reply:
added validation